internal/server/otel/attributes.go

```diff
@@ -0,0 +1,15 @@
+package otel
+
+import "go.opentelemetry.io/otel/attribute"
+
+var (
+	AttributeMatch       = attribute.Key("flipt.match")
+	AttributeFlag        = attribute.Key("flipt.flag")
+	AttributeFlagEnabled = attribute.Key("flipt.flag_enabled")
+	AttributeSegment     = attribute.Key("flipt.segment")
+	AttributeReason      = attribute.Key("flipt.reason")
+	AttributeValue       = attribute.Key("flipt.value")
+	AttributeEntityID    = attribute.Key("flipt.entity_id")
+	AttributeRequestID   = attribute.Key("flipt.request_id")
+	AttributeDurationMS  = attribute.Key("flipt.duration_ms")
```

Comment:
Food for thought: going forward I wonder if we can reconcile these: https://github.com/flipt-io/flipt/blob/main/internal/server/metrics/metrics.go#L54-L60

For now, I dont think it is worth it. Just a shame as there is some overlap.
I wish I had namespaced those `metrics.go` ones like you have.

Reply:
yeah i thought about that, the only difference right now is the namespace I think